roles/deploy/tasks/deploy.yml

```diff
@@ -73,6 +73,16 @@
     - skip_ansible_lint
   notify: restart unicorn
 
+# Install JS dependencies with yarn
+
+- name: run yarn install
+  yarn:
+    path: "{{ current_path }}"
+    executable: "/usr/local/lib/npm/bin/yarn"
```

Comment:
after provisioning, I connect to the server as openfoodnetwork and yarn is on the PATH and I can run "yarn install". I wonder why is this task not picking it up?
Without this executable config it errors out saying it cant find the yarn executable:
`Failed to find required executable yarn in paths: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin"`

Reply:
Ansible executes things in a slightly odd way with Python modules, and some of the environment bits like PATH of the specified user are often not as expected within the context of a task.

Specifying the executable looks fine here :+1:

Reply:
ok, thanks!